scipy/linalg/tests/test_decomp_polar.py

```diff
@@ -91,3 +92,29 @@ def test_verify_cases():
     for a in verify_cases:
         verify_polar(a)
 
+@pytest.mark.parametrize('dt', [int, float, np.float32, complex, np.complex64])
+def test_empty(dt):
+    empty_cases = [
+    np.empty((0, 0), dt),
+    np.empty((0, 2), dt),
+    np.empty((2, 0), dt),
+    ]
+
+    for a in empty_cases:
```

Comment:
I'll go ahead and `@parametrize(shape, [(0, 0), (0, 2), (2, 0)])` to avoid the explicit loop.